.ci/bump-cs-version-in-sonar-plugin.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+if [[ -z $1 ]]; then
+  echo "ERROR: version is not set"
+  echo "Usage: bump-cs-version-in-gradle-project.sh <version>"
+  exit 1
+fi
+
+VERSION=$1
+SONAR_PLUGIN_FILE=sevntu-checkstyle-sonar-plugin
```

Comment:
file is pom.xml

Reply:
Ya but the project has only one pom file so it works well without adding `pom.xml`

Reply:
see comment below

Reply:
Changes Done